style.css

```diff
@@ -620,139 +620,211 @@ input:valid {
   justify-content: center;
 }
 
-#pop-outer {
-  height: 836px;
-  width: 375px;
-  width: max-content;
-  background-color: #fff;
-  border-radius: 16px;
+.mobile-cards {
+  display: grid;
   padding: 16px;
-  margin-left: 16px;
-  margin-top: 200px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}

```

Comment:
- [ ] In the Figma template there is a light grey container around the modal window, please add it to be similar to the Figma template 👇🏽 
- [ ] It is evident that the pop-up window is bigger than the screen size in that case we could use `overflow-y: scroll;` please 🙏🏽 fix it to be similar to the Figama template, the image should show complete for a better UI.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/232225858-d3067e77-4ab8-4570-87d3-a0b00368c5e2.png) | ![image](https://user-images.githubusercontent.com/58642949/232225920-61604c59-9890-443c-ba3e-cf21342bc522.png) |
| ![image](https://user-images.githubusercontent.com/58642949/232225968-2c0848a1-7e3b-4f4c-8ea9-e952e47c3da7.png) | ![image](https://user-images.githubusercontent.com/58642949/232226041-90b9ea6e-93cf-42cc-8413-db62b8d43c5d.png) |